spec/CU-86ewu2krt/v1/plans/plan-2.md

```
@@ -0,0 +1,130 @@
+nix run .#md-mermaid-lint -- --version
+
+# Test with sample markdown
+echo '```mermaid
+graph TD; A-->B
+```' > /tmp/test.md
+nix run .#md-mermaid-lint -- /tmp/test.md
+````
+
+## Expected Output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)